### DISCUSSION TITLE
Entering ".com" is frustrating

### DISCUSSION THREAD

**Discussion by jshir** [AnySoftKeyboard/AnySoftKeyboard#3820] [Q&A]
First of all sorry to show up and be high maintenance. I am coming from the original Swype keyboard that Android 14 has finally completely killed off. I use swipe gestures for everything I possibly can.

I am finding it hard to enter .com which is something I do fairly often. On Swype, I could start my gesture on the period and swype through c o m to get ".com" in one swipe. That doesn't work in AnySoftKeyboard.

Frustratingly, if I try typing the period and then swiping "com" it puts a space between the . and com like ". com" which is not what I want.

I am not creating an issue for this because I assume I may just be using my new keyboard wrong, is there a way to input .com nicely?

**Comment by skroobs4** [comment]
In the user interface(ui) make sure all the specialized input fields are checked, there is a URL input field that provides a .com key

**Reply by jshir** [reply]
That was already checked, but I hadn't noticed the .com key because of all the types of fields I need to type ".com" it's rare that it is a proper URL input field. I quite liked having the swipe gesture for .com, is that something that may be added to this keyboard?

**Reply by SwiftScriptr** [reply]
It's nice to know I'm not alone in this; I shall similarly explain an annoying scenario. 
Firstly, let me disclose that all automated speech inputs are turned off:
No complete end of the word,
No scan word length suggestions etc.

I was not aware of this issue until I frequently used a terminal emulator on android 8, (not that I think it matters).
Anyone who has heard the concept of Unix file navigation understands that going back a directory requires a double dot '..'
However, if I am not selecting the full stop key at a snails pace, whenever I type 'cd' (change directory) and a period to signify/start a double dot (the previous directory), the full-stop will jump across to the letters and leave a gap like so 'X. .' where x could be any character the screen. 

Could it be that this is a predetermined auto correct no matter what the settings are?

If so, no matter what the: UI, key-gestors, Theme/layout even Android ver, this clearly shows an indiscriminate overriding of what the author wants to type. I like the app but it is a 'patience building' exercise when using a command line that requires constant navigation.

Most layouts of a keyboard have the period/full-stop (depending on what you call it) next to the space bar so I believe it's an inevitable issue experienced by most users. Meaning nearly all themes (my selected one having it on the right side of the board) will have this issue If and when explored with typing speeds. It is literally a goldilocks issue where if you don't type intentionally, it autocorrects.

I understand that you may not be a dev, but I hope this raises awareness for them to solve this last  issue with the app IMO.

**Comment by The-Dolphi** [comment]
Not yet, Swype is still alive xD

Some guy called KillaMD released a mod that works on Andoid 14:
https://xdaforums.com/t/any-way-to-run-swype-dragon-keyboard-on-android-14-on-pixel-6-pro.4640113/
